ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/ValidatedCommandFactory.java

```diff
@@ -92,17 +95,17 @@ private static Command ensureDeserializable(final Command command) {
     }
   }
 
-  private static Command createCommand(
+  private static Optional<Command> createCommand(
```

Comment:
There's something I don't feel good about this 'cause you create a command but it may return nothing. This is breaking the meaning of the method. Is there something else we can do?